### DISCUSSION TITLE
Window grey out when logging into Celeste

### DISCUSSION THREAD

**Discussion by chrismacor** [hwittenborn/celeste#160] [General]
I'm attempting to sync with Google drive on Jellyfish.
When I click login the window greys out and nothing happens.
Any suggestions?
Thanks
Christopher

**Comment by hwittenborn** [comment]
Hey @chrismacor, see #161 for the tracking of this issue.

I usually use the issues pages (in addition to the Discord support server) for support requests - I've been meaning to close up the Discussions page but I haven't gotten to it quite yet. For now though let's continue the conversation there.
